### PR TITLE
chore: merge `dev` branch to `main` branch

### PR DESCRIPTION
**Pull request by EvadeMaster**
## Merge `dev` branch to `main` branch

Prerequisite before merge

- [x] Pull Request mark as ready (by author or the ReVanced Manager Team)
- [x] Approved by the Manager Team
- [x] Bump version to v1.3.9

--------------------

16 Commits (1+4+3+5+2+1)

* ✨ 1 Feature
* 🐛 4 Bug Fixes
* ⚙️ 3 Build Changes
* ♻️ 5 Code Refactoring
* 📝 2 Documentation Improvements
* 🧹 1 Miscellaneous Change

--------------------

ReVanced Manager v1.3.9 (Next)

Experienced in Kotlin? Consider checking out [ReVanced Manager Compose](https://github.com/revanced/revanced-manager-compose)!

### CODE REVIEW

**Review by TheAabedKhan — CHANGES_REQUESTED** [review]
No semicolon on 119 and 120??

File: lib/ui/widgets/settingsView/settings_export_section.dart [hunk 1]
```diff
@@ -91,9 +92,36 @@ class SExportSection extends StatelessWidget {
...
+          CustomMaterialButton(
+            isFilled: false,
+            label: I18nText('noButton'),
+            onPressed: () => Navigator.of(context).pop(),
+          ),
+          CustomMaterialButton(
+            label: I18nText('yesButton'),
+            onPressed: () => {
+              Navigator.of(context).pop(),
+              _settingsViewModel.resetSelectedPatches()
```
[the remark this answers is not in the document]
Comment (EvadeMaster) [reply lib/ui/widgets/settingsView/settings_export_section.dart:120]: Hmm? What about it, the code seems to work fine as intended.

[the remark this answers is not in the document]
Comment (TheAabedKhan) [reply lib/ui/widgets/settingsView/settings_export_section.dart:120]: Oh, I missed the `=>`

**Review by TheAabedKhan — APPROVED** [review]
This looks great ✨

**Pull request merged**